dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -119,6 +119,16 @@
   GHERKIN
 end
 
+Given(/^I am a program manager with started and completed courses$/) do
```

Comment:
Added this to make the Workshops table show up on the `Regional Partner Center` tab. It's similar to the one above.